app/models/xccdf/benchmark.rb

```diff
@@ -29,6 +29,16 @@ def from_openscap_parser(op_benchmark)
       def policy_class
         BenchmarkPolicy
       end
+
+      def latest
+        all.map(&:ref_id).uniq.map { |ref_id| find_latest(ref_id) }
```

Comment:
Let's try to leverage the DB as much as possible.

```suggestion
        select(:ref_id).distinct.find_each.map { |ref_id| find_latest(ref_id) }
```